jsk_fetch_robot/jsk_fetch_startup/launch/fetch_insta360_indigo.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
+  <!-- Insta360 images -->
+  <include file="$(find jsk_perception)/sample/sample_insta360_air.launch">
+    <arg name="gui" value="false" />
+    <!-- Reduce CPU load -->
+    <!-- To check the valid formats, -->
+    <!-- $ sudo apt-get install v4l-utils -->
+    <!-- $ v4l2-ctl -d /dev/insta360 -\-list-formats-ext -->
+    <arg name="height" value="736" />
+    <arg name="width" value="1472" />
+    <!-- for indigo -->
+    <arg name="use_usb_cam" value="true" />
+  </include>
+  <!-- Split insta360 air image into 2 fisheye image -->
+  <!-- This node can be removed if below is merged -->
+  <!-- https://github.com/jsk-ros-pkg/jsk_recognition/pull/2555 -->
+  <node pkg="jsk_perception" name="split_image" type="split_image.py" output="screen">
+    <remap from="~input" to="/insta360/image_raw" />
+    <rosparam>
+      vertical_parts: 1
+      horizontal_parts: 2
+      rate: 5
+    </rosparam>
+  </node>
```

Comment:
@708yamaguchi so you mean we dont need this node?

Reply:
Yes, this is because `split_image.py` is launched in https://github.com/jsk-ros-pkg/jsk_recognition/blob/cce4067b0d1ea9e1ab8decfeda9d1a8da07c8c61/jsk_perception/sample/sample_insta360_air.launch#L51-L58

```
  <!-- Split insta360 air image into 2 fisheye image -->
  <!-- This node can be removed if below is merged -->
  <!-- https://github.com/jsk-ros-pkg/jsk_recognition/pull/2555 -->
```

Reply:
OK, so how about these lines?
do we need these lines?

https://github.com/jsk-ros-pkg/jsk_robot/blob/b16550112790999940c61b65996731b4917ba9a4/jsk_fetch_robot/jsk_fetch_startup/launch/fetch_insta360_indigo.launch#L15-L28

Reply:
Thank you.

I created PR https://github.com/jsk-ros-pkg/jsk_robot/pull/1394

Reply:
Good!  Thanks.